test/integration/freetext_editor_spec.js

```diff
@@ -752,7 +753,7 @@ describe("FreeText Editor", () => {
               "switchannotationeditorparams",
               {
                 source: null,
-                type: /* AnnotationEditorParamsType.FREETEXT_SIZE */ 1,
+                type: /* AnnotationEditorParamsType.FREETEXT_SIZE */ 11,
```

Comment:
Can we avoid hard-coding this?
```suggestion
                type: window.pdfjsLib.AnnotationEditorParamsType.FREETEXT_SIZE,
```